webpack.config.js

```diff
@@ -101,7 +101,7 @@ module.exports = {
       {
         from: './meinberlin/assets/images/**/*',
         to: 'images/',
-        flatten: false
+        flatten: true
```

Comment:
It puts all files from all sub directories into the target without any additional directories below it.